### PR TITLE
WEBUI-875: edition of a comment containing non-alpha-numeric characters

### PR DESCRIPTION
**Pull request by rakeshkumar1019**
https://jira.nuxeo.com/browse/WEBUI-875

### CODE REVIEW

**Review by GuruprasadAnk1996 — DISMISSED** [review]

**Review by SnehalNaik-DEV — DISMISSED** [review]
Looks Good.

**Review by swarnadipa-dev — DISMISSED** [review]
Looks good

File: elements/nuxeo-document-comments/nuxeo-document-comment-thread.html [hunk 1]
```diff
@@ -276,6 +276,7 @@
           this.$.selector.select(comment);
           this.set('reply', comment.text);
           this.$$('#replyContainer').focus();
+          this.reply = event.detail.comment;
```
Comment (Gabez0r) [inline elements/nuxeo-document-comments/nuxeo-document-comment-thread.html]: This makes line 277 redundant. Also, why are we not doing the same for LTS 2021 (https://github.com/nuxeo/nuxeo-elements/pull/599/)?

Comment (rakeshkumar1019) [reply elements/nuxeo-document-comments/nuxeo-document-comment-thread.html]: @Gabez0r made the changes. Thank you.

**Review by swarnadipa-dev — APPROVED** [review]
Looks good

**Review by SnehalNaik-DEV — APPROVED** [review]
Looks good

**Review by GuruprasadAnk1996 — APPROVED** [review]

**Review by Gabez0r — APPROVED** [review]

**Review by richardsd — APPROVED** [review]

**Review by rajatgoel-gl — APPROVED** [review]

**Pull request merged**